glide.yaml

```diff
@@ -1,8 +1,7 @@
 package: go.uber.org/fx
 import:
-- package: github.com/uber-go/zap
-  # Pinned temporarily while zap reaches 1.0 (https://github.com/uber-go/fx/pull/243)
-  version: c064b5c44b285a7e2fd5c9b26e8c38228ce2bccb
+- package: go.uber.org/zap
+  version: v1.0.0-rc.1
```

Comment:
Typical for an RC - depends on how many things I got wrong :)

I'm hoping to update some other internal projects this week and cut a 1.0 final before the end of the month. The APIs for the loggers themselves are very, very unlikely to change; more likely that some portion of the constructors changes. 

I misspelled a JSON key, so there's and RC.2 coming Monday. Shouldn't affect you guys.